hack/test-sh.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+#
+# Copyright 2019 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+REPO_ROOT=$(git rev-parse --show-toplevel)
+cd "${REPO_ROOT}"
+find . -type f -name '*_test.sh' -print0 | xargs -0 bash
```

Comment:
Same as before, this calls
```
bash 00_test.sh 01_test.sh 02_test.sh
```
and not
```
bash 00_test.sh
bash 01_test.sh
bash 02_test.sh
```

One thing that would do what you want with find & xargs is:
```
find . -type f -name '*_test.sh' -print0 | xargs -0 -n1 /usr/bin/env bash
```

Reply:
Oh okay, I don't think I understood what you were saying before. I changed it to be one argument at a time